barman/retention_policies.py

```diff
@@ -170,6 +193,12 @@ def _backup_report(self, source):
         # NOTE: reverse key orders (simulate reverse chronology)
         i = 0
         for bid in sorted(backups.keys(), reverse=True):
+            if backups[bid].is_incremental:
+                _logger.debug(
+                    "Ignoring incremental backup %s from retention policy report."
```

Comment:
I wonder if this log message will sound confusing to whoever reads the log later?
I mean, we are indeed ignoring this in the main loop, but that's because we are going to handle this through other means.

I wonder if we have better ways to express that, or if we should leave it as is -- or remove it. Thoughts?

Reply:
```
_logger.debug(
                    "Ignoring incremental backup %s. It will have the retention"
                    " policy status propagated from %s."
                    % (backups[bid], backups[bid].parent_backup_id)
                )
```
OR 

```
_logger.debug(
                    "Ignoring incremental backup %s. The retention status will"
                    " be propagated from %s if parent status is VALID,"
                    " OBSOLETE, POTENTIALLY OBSOLETE. If parent has any KEEP*"
                    " status, incremental backup will be VALID."
                    % (backups[bid], backups[bid].parent_backup_id)
                )
```
@barthisrael @gcalacoci

Reply:
I think the first option should be ok for now.
If we think this is flooding the logs, we may choose to remove this message too.
I'm interested in @gcalacoci 's thoughts later as well.